Directory.Build.targets

```diff
@@ -5,6 +5,13 @@
 
   <Import Project="$(RepositoryEngineeringDir)testing\runsettings.targets" Condition="'$(IsTestProject)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)placeholderpackaging.targets" Condition="'$(IsPlaceholderTargetFramework)' == 'true'" />
+
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true' and '$(IsSourceProjectPackable)' == 'true'">
+    <!-- IsSourceProjectPackable is used in the CI packing leg to enable IsPackable only on source projects.
+         This needs to be evaluated before importing packaging.targets. -->
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
```

Comment:
The Arcade SDK might also condition on `IsPackable` so it would be better to move this further up before all the Imports.

Reply:
Moved.